### PR TITLE
Added documentation for the _sender property

### PR DESCRIPTION
**Pull request by durandj**
Also bumped the version to 1.1.0

### CODE REVIEW

**Comment by durandj** [comment]
This should address #43

File: README.md [hunk 1]
```diff
@@ -118,6 +118,42 @@ class ContentApp extends Component {
 }
 ```
 
+### 4. Optional: Retrieve information about the initiator of the action
+
+There are probably going to be times where you are going to want to know who sent you a message. For example, maybe you have a content script and you want to have it send information to a store that is managed by the background script and you want your background script to know which tab sent the information to it. You can retrieve this information by using the `_sender` property of the action. Let's look at an example of what this would look like.
```
Comment (vhmth) [inline README.md]: I believe (IIRC) that @tshaddix prefers using UI Component in place of "content script" wherever possible. (correct me if I'm wrong Tyler)

This looks dope though!

File: README.md [hunk 2]
```diff
@@ -118,6 +118,42 @@ class ContentApp extends Component {
...
+        return Object.assign({}, ...state, {
+            lastTabId: action._sender.tab.id
+        });
+    default:
+        return state;
+    }
+}
+```
+
+No changes are required to your actions, instead react-chrome-redux adds the information for you when you use a wrapped store.
```
Comment (vhmth) [inline README.md]: Can you remove "instead" and make it "react-chrome-redux automatically adds this information"?

**Comment by vhmth** [comment]
Overall looks good to me.

**Comment by vhmth** [comment]
I'll let @tshaddix chime in.

File: README.md (hunk 2 above)
Comment (durandj) [inline README.md]: No problem!

File: README.md (hunk 1 above)
Comment (durandj) [inline README.md]: Sure I can make that change.

**Comment by vhmth** [comment]
@tshaddix friendly ping

**Comment by tshaddix** [comment]
Sorry @durandj and @vhmth. Just got back from vacation. I'll merge now and bump. Thanks everyone!

**Pull request merged**